public/pages/ReviewAndCreate/components/DetectorDefinitionFields/DetectorDefinitionFields.tsx

```diff
@@ -88,8 +88,10 @@ export const DetectorDefinitionFields = (
         />
       );
     }
-    // if validationResponse is not undefined and there was no error from validation display one
-    // of the callout options.
+    // Callouts only displayed based on if validDetectorSettings is true or not, referring to
+    // to response content from validation API (empty body or response issue body).
+    // validationError reffers to if there was an exception from validation API which means no callout will
```

Comment:
`reffers` -> `refers`. Also, can you add an example on what a possible exception could be? (security, failure reading the index, etc.).